### PR TITLE
assaultcube: init at official master branch , rev = "9f537b0876a39d7686e773040469fbb1417de18b"

### PR DESCRIPTION
**Pull request by bignaux**
###### Motivation for this change

fun first-person-shooter, need to build on git since latest official release is out of date (2013) ...

###### Things done

Tested only on my non-nixos OS (gentoo ~amd64) . I've (so common) gl issues , but this old good games since to work well.

### CODE REVIEW

File: pkgs/games/assaultcube/default.nix [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{ fetchgit, stdenv, makeDesktopItem, curl, enet, openal, pkgconfig, libogg,
```
Comment (srhb) [inline pkgs/games/assaultcube/default.nix]: curl and SDL_mixer are never used, as far as I can see.

File: pkgs/games/assaultcube/default.nix [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  libvorbis, SDL, SDL_image, SDL_mixer, makeWrapper, zlib,
+  client ? true, server ? true }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  # master branch has legacy (1.2.0.2) protocol 1201 and gcc 6 fix.
+  branch = "master";
```
Comment (srhb) [inline pkgs/games/assaultcube/default.nix]: Could we have some sort of version attribute here, appended to name? Perhaps just the date of the commit in question.

File: pkgs/games/assaultcube/default.nix [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  meta = {
+    description = "Fast and fun first-person-shooter based on the Cube fps";
+    homepage = http://assault.cubers.net;
+    maintainers = [ maintainers.genesis ];
+    license = stdenv.lib.licenses.zlib;
+  };
+
+  #sadless , fetchgit seems to refetch all stuff when man switch branch .
+  src = fetchgit {
```
Comment (srhb) [inline pkgs/games/assaultcube/default.nix]: I think fetchFromGitHub would be preferable here.

File: pkgs/games/assaultcube/default.nix [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    url = "https://github.com/assaultcube/AC.git";
+    rev = "9f537b0876a39d7686e773040469fbb1417de18b";
+    branchName = "${branch}";
+    #sha256 = "";
```
Comment (srhb) [inline pkgs/games/assaultcube/default.nix]: Needs a hash!

**Comment by srhb** [comment]
@GrahamcOfBorg build assaultcube

**Comment by bignaux** [comment]
@GrahamcOfBorg build assaultcube

**Comment by srhb** [comment]
@GrahamcOfBorg build assaultcube

**Pull request merged**